tests/integration/IVADefects/M1taxonomy/steps/returnAllDefectsStep.ts

```diff
@@ -0,0 +1,51 @@
+import {Given, When, Then} from "@cucumber/cucumber"
+import chai from "chai"
+import { generateToken } from "./common"
+import * as expectedJson from "../data/m1taxomomy.json";
+const {expect}=chai;
+var token : string;
+
+var baseURL="https://api.integration.cvs.dvsacloud.uk/integration/v3/"
```

Comment:
Would this be better in a secret or env variable perhaps? As will it not be different when in develop?